drivers/hmis/app/jobs/hmis/auto_exit_job.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis
+  class AutoExitJob < BaseJob
+    def perform
+      Hmis::Hud::Project.all.each do |project|
+        config = Hmis::AutoExitConfig.config_for_project(project)
+        next unless config.present?
+
```

Comment:
Can we add in a safeguard here, like `raise 'Auto-exit config unusually low: {days}' if config. length_of_absence_days < 30`?  I worry about some config getting created with something like 0 days, which would cause problems and shouldn't happen.

Reply:
This too, just to be safe